src/Normalizer/MoocNormalizer.php

```diff
@@ -75,14 +77,14 @@ private function normalizeElement(BaseMoocElement $element): array
             'attachments' => $this->normalizeFiles($element->getFiles()),
         ];
 
-        if (BaseMoocElement::ELEMENT_TYPE_VIDEO === $element->getType()) {
+        if ($element instanceof Video) {
```

Comment:
You can now use `if ($element->isVideo()) {`

Reply:
If I use this I will have to add a doc bloc for ide understand wich class is it ^^,